onboard/cv/camera.cpp

```diff
@@ -2,9 +2,18 @@
 #include "perception.hpp"
 
 #if ZED_SDK_PRESENT
+
+#pragma GCC diagnostic ignored "-Wreorder" //Turns off warning checking for sl lib files
```

Comment:
There's no way around this? Seems like Travis might be ignoring it

Reply:
also-- is it failing on warning checking? Or are we just disabling it because warning checking is messy/looks bad?

Reply:
We're just doing it because it looks bad. There are a few camera resolution warnings that are thrown that I have been having a hard time getting rid of that you're probably seeing on Travis.